Makefile

```diff
@@ -233,12 +233,20 @@ package: clean check_bundledir $(BUILD_DIR)/macos-amd64/crc $(HOST_BUILD_DIR)/cr
 	chmod 755 packaging/darwin/scripts/postinstall
 	mkdir -p packaging/root/"$(MACOS_INSTALL_PATH)"/$(CRC_VERSION)/
 	$(HOST_BUILD_DIR)/crc-embedder download packaging/root/"$(MACOS_INSTALL_PATH)"/$(CRC_VERSION)/
+
+	tar -C packaging/root/"$(MACOS_INSTALL_PATH)"/$(CRC_VERSION)/ -xvzf packaging/root/"$(MACOS_INSTALL_PATH)"/$(CRC_VERSION)/crc-tray-macos.tar.gz
```

Comment:
If I'm not mistaken, we'll have a `packaging/root/Library/1.21.0/` directory which contains
- `crc`
- `crc_hyperkit_4.6.9.crcbundle`
- `CodeReady Containers.app`

It's imo a weird tree structure, we have a .app, but we don't have everything crc-related in it.

Reply:
Yes my plan is to invert this in a follow-up:
/Applications/CodeReady Containers.app with a Resources directory containing crc, bundle, etc.

With that, for uninstalling, we can tell the user to remove ~/.crc and drag and drop the app in the trash.
It also feels like other apps.